.github/workflows/docsbuild-release.yaml

```
@@ -0,0 +1,78 @@
+name: ansible-release-porting-guide
+on:
+  workflow_dispatch:
+    inputs:
+      ansible-version:
+        description: >-
+          Release Version. Example: 11.1.0
+        required: true
+env:
+  CI_COMMIT_MESSAGE: 'Add Ansible community "${{ inputs.ansible-version }}" porting guide'
```

Let's consistently avoid quotes in YAML strings so the style wouldn't be arbitrary across the file:
```suggestion
  CI_COMMIT_MESSAGE: >-
    Add Ansible community "${{ inputs.ansible-version }}" porting guide
```